content/en/blog/2023/steering-contribution-seat-results/index.md

```diff
@@ -0,0 +1,21 @@
+---
+title: "2023 Istio Steering Contributor Seats"
+subtitle: Announcing the Istio Steering Committee Contributor Members, 2023.
+description: Announcing the Istio Steering Committee Contributor Members, 2023.
+publishdate: 2023-02-04
+attribution: "Faseela K, for the Istio Steering Committee"
+keywords: [istio,steering,governance,community,contributor]
+---
+
+Based on our [Steering Committee charter](/blog/2020/steering-changes/), Istio shares governance responsibilities between Contribution Seats, selected based on contributions to the project, and Community Seats, elected by the project members. The [Istio Steering Committee](https://github.com/istio/community/blob/master/steering/README.md) consists of 9 proportionally-allocated Contribution Seats, and 4 elected Community Seats.
+
+Last year, we [elected four members](/blog/2022/steering-election-results/) to the community seats, with the committee representing eight different companies. It's now time to reward the companies who fuel our growth with the maximum contributions to Istio, by selecting the [Contributor seat](/blog/2020/steering-changes/#contribution-seats) members to the Istio Steering Committee. As per the [Steering charter](https://github.com/istio/community/blob/master/steering/CHARTER.md), the Contribution Seat numbers are calculated in February, and the last agreed [metric](https://github.com/istio/community/blob/master/steering/CONTRIBUTION-FORMULA.md) is Contributions, as measured by [devstats](https://istio.devstats.cncf.io/d/66/developer-activity-counts-by-companies?orgId=1&var-period_name=Last%20year&var-metric=contributions&var-repogroup_name=All&var-country_name=All&var-companies=All).
```

Comment:
"reward" is a strange term to use.  (I think we normally also say 7, counting IBM/Red Hat as one group.)

```suggestion
Last year, we [elected the four members](/blog/2022/steering-election-results/) to the community seats. It's now time to reward the companies who fuel our growth by selecting the [Contribution Seat] members. As per the [Steering charter](https://github.com/istio/community/blob/master/steering/CHARTER.md), every February we look at which companies have made the most contributions to Istio based on an [annually agreed metric](https://github.com/istio/community/blob/master/steering/CONTRIBUTION-FORMULA.md).
```